documentation/Functions.md

```diff
@@ -15,6 +15,7 @@ The following functions are already implemented:
 * [cloneRepository](#cloneRepository)
 * [runServerJava](#runServerJava)
 * [buildNg](#buildNg)
+* [runClientNg](#runClientNg)
```

Comment:
We don't have an order of the functions at the moment. I think it would be better to arrange the functions in alphatetic order. What do you think?

Reply:
Yes, I was working on that part only. I will modify it in Alphabetical order.